packages/language-service/lib/commands.js

```diff
@@ -146,3 +139,12 @@ export function createSyntaxToggle(context, type, separator) {
     }
   }
 }
+
+export const implementations = {
+  'mdx.toggleDelete': createSyntaxToggle('delete', '~'),
```

Comment:
where’s this word from, `delete`? Perhaps `strikethrough` is better. Also, it’s a GFM-only feature. And, 2 tildes work there too. One tilde was forbidden by the GFM spec for a while, until recently, even though it worked.

Reply:
`delete` is the mdast Node type. Based on your comment, I agree inserting 2 tildes is better. This changed be done in a follow-up PR though, as it’s an unrelated change.

Reply:
This doesn’t particularly have to match the node type, as it’s injecting into a document.

Reply:
It’s a toggle. It either injects or removes, depending on whether the cursor position matches a node of this type.